tests/cisco_nxos/show_ip_interface/cisco_nxos_show_ip_interface1.raw

```diff
@@ -0,0 +1,31 @@
+IP Interface Status for VRF "default"

```

Comment:
While not impossible, this does not seem likely like a raw output of the command (based on the other commands). I am going to guess there is something else in your process that is stripping the whitespace before the lines. Can you confirm if that is the case or not?

Reply:
I'm sorry the changes on NXOS files were taken from https://github.com/networktocode/ntc-templates/pull/1046
Initially, testing my commit, I had some issues on Cisco files I didn't changed. Few days later I synced the master repo again and tests completed successfully without other changes. In that moment, probably, I made some errors including the commit from 1046. Let me know if you can still merge or if I should make another PR.
My PR involves HP Procurve files only.

Reply:
@jvanderaa, @itdependsnetworks any thoughts? Should I make another PR?